app/src/main/res/navigation/navigation.xml

```diff
@@ -102,12 +99,13 @@
             android:id="@+id/action_userProfileFragment_to_reportFragment"
             app:destination="@id/reportFragment"
             app:popUpTo="@id/itemsListFragment" />
+        <action
+            android:id="@+id/actionuserProfileFragmentToItemsListFragment"
+            app:destination="@id/itemsListFragment"
+            app:popUpTo="@id/userProfileFragment"
+            app:popUpToInclusive="false" />
```

Comment:
What does this do?

Reply:
`popUpTo` is something that "pops" the fragments' stack. Meaning, if we have the following navigation:
`ItemsList -> ClickOnItem1 -> ClickOnUsername -> UserProfileFragment -> Report` then, when you click on `<-`, you will get the desired behaviour `Report -> UserProfileFragment -> DetailedItemView -> ItemList`. However, as I did for the Report feature, if you set PopUpTo to `ReportFragment`, it means that when you click on `<-` from the UserProfileFragment, instead of going to `ReportFragment` you will pop the stack and just go back to the ItemList. This is useful if we don't want to be able to come back to a previous fragment (i.e. when I reported someone, by clicking on `<-` without having popped, I would be able to come back to the ReportFragment and report a user again, or even have an udnefined behaviour if the arguments are not set up properly). If I understood the documentation properly, `popUpToInclusive` means whether we stop at the fragment we are popping up to, or do we also pop the fragment who is in the "popUpTo".

Reply:
Understandable, thanks